backend/src/main/java/moaon/backend/global/config/WebConfig.java

```diff
@@ -10,14 +10,15 @@ public class WebConfig implements WebMvcConfigurer {
     @Override
     public void addCorsMappings(CorsRegistry registry) {
         registry.addMapping("/projects/**")
-                .allowedOrigins(
-                        "http://localhost:3000",
-                        "https://moaon-dev.vercel.app",
-                        "https://d2ye9egv48bag1.cloudfront.net"
-                )
-                .allowedMethods("GET")
-                .allowedHeaders("*")
-                .allowCredentials(true)
-                .maxAge(3600);
+            .allowedOrigins(
+                "http://localhost:3000",
+                "https://moaon-dev.vercel.app",
+                "https://d2ye9egv48bag1.cloudfront.net",
+                "https://moaon-preview-*.vercel.app"
+            )
```

Comment:
```suggestion
            .allowedOrigins(
                "http://localhost:3000",
                "https://moaon-dev.vercel.app",
                "https://d2ye9egv48bag1.cloudfront.net",
            )
        .allowedOriginPatterns("https://moaon-preview-*.vercel.app")
```

경로 와일드카드를 인식하게 하려면 위처럼 `allowedOriginpatterns` 메서드를 써야한다고 하네욥

Reply:
오 몰랐어요 감사해요!
3700bca

[CORS 사양](https://fetch.spec.whatwg.org/#cors-protocol-and-credentials)에 따라 클라이언트의 쿠키/인증정보 포함 요청을 허용할 때에는 명확한 출처를 지정해야만 하기 때문이라고 하네요. `allowedOriginPatterns`를 이용해 스프링에서 제한적으로 명확한 출처로 지정함으로서 와일드카드를 사용할 수 있다고 하네요.